lib/shipit.rb

```diff
@@ -75,7 +75,15 @@ def redis_url
   end
 
   def redis(namespace = nil)
-    @redis ||= Redis.new(url: redis_url.to_s.presence, logger: Rails.logger)
+    @redis ||= begin
+      Redis.new(
```

Comment:
style: you can get rid of the begin/else:

```ruby
@redis ||= Redis.new(
  url: ....,
  foo: ...,
)
```

Reply:
good catch, thanks!